src/modules/dashboard/components/CreateColonyWizard/StepSelectToken.tsx

```diff
@@ -71,8 +81,17 @@ const MSG = defineMessages({
   },
 });
 
+const isXdai = DEFAULT_NETWORK_TOKEN.symbol === XDAI_TOKEN.symbol;
+
 export const validationSchema = yup.object({
-  tokenAddress: yup.string().address(() => MSG.invalidAddress),
+  tokenAddress: yup
+    .string()
+    .address(() => MSG.invalidAddress)
+    .test(
+      'is-not-addressZero',
+      () => (isXdai && MSG.xdaiAddress) || MSG.ethAddress,
```

Comment:
So what happens when we add another network to the mix, one that is not ethereum and not xdai?

See my above suggestion about passing in the `DEFAULT_NETWORK_TOKEN` as a value to the message descriptor.

I think you'll need to use `formattedMessage()` from `react-intl` for this

Reply:
i also wanted to do like that, but like you see, yup object schema is define outside of the component. And to use formatMessage method we need to use hook useUntil(). I was also trying to return component <FormattedMessage /> but it also wasn't work